http/src/main/resources/init-resources/startup.sh

```diff
@@ -121,6 +121,11 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
     # update container MEM_LIMIT to reflect VM's MEM_LIMIT
     docker update $JUPYTER_SERVER_NAME --memory $MEM_LIMIT
 
+    # See IA-1901: Jupyter UI stalls indefinitely on initial R kernel connection after cluster create/resume
```

Comment:
I wonder if we should revert the previous PR that changed the proxy service if that didn't really fix things 🤔

Reply:
Yeah seems reasonable.. I reverted it

Reply:
oh cool..did't see `||`